src/test/java/hudson/tasks/AntTest.java

```diff
@@ -141,9 +162,9 @@ public void testParameterExpansion() throws Exception {
                 "vNUM=$BUILD_NUMBER\nvID=$BUILD_ID\nvJOB=$JOB_NAME\nvTAG=$BUILD_TAG\nvEXEC=$EXECUTOR_NUMBER\n"
                 + "vNODE=$NODE_NAME\nvLAB=$NODE_LABELS\nvJAV=$JAVA_HOME\nvWS=$WORKSPACE\nvHURL=$HUDSON_URL\n"
                 + "vBURL=$BUILD_URL\nvJURL=$JOB_URL\nvHH=$HUDSON_HOME\nvJH=$JENKINS_HOME\nvFOO=$FOO\nvAX=$AX"));
-        assertBuildStatusSuccess(project.scheduleBuild2(0, new UserCause()));
+        r.assertBuildStatusSuccess(project.scheduleBuild2(0, new UserCause()));
         MatrixRun build = project.getItem("AX=is").getLastBuild();
-        String log = getLog(build);
+        String log = JenkinsRule.getLog(build);
         assertTrue("Missing $BUILD_NUMBER: " + log, log.contains("vNUM=1"));
```

Comment:
I have not addressed this since `assertLogNotContains` will load the logs every time it is called which would be time consuming since there would be many calls, but mainly because the error message is probably better as it is rather than the one that `assertLogNotContains` will offer.